src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -18,6 +18,10 @@ CaptureViewElement::CaptureViewElement(CaptureViewElement* parent, TimeGraph* ti
 void CaptureViewElement::SetWidth(float width) {
   if (width != width_) {
     width_ = width;
+
+    for (auto& child : GetChildren()) {
+      child->SetWidth(width);
```

Comment:
Should we add a comment saying that children in general occupt all the width their parents gave them? Because if not this seems a bit strange. More generic could be modify children's width proportionally.

Reply:
Yes, that's reasonable, I've added the comment. We may want to change the behavior at some point.